### PR TITLE
[dev] show battery reminder by easy to see GUI

### PR DESCRIPTION
**Pull request by HarukiKozukapenguin**
I develop new [GUI](https://drive.google.com/file/d/1vjq5bNMCUjl4o5YUgqSbA79bBDmIaRcb/view?resourcekey) (thanks by @KenMat765) to show battery reminder for JSK aerial robotics team.

### CODE REVIEW

**Comment by sugikazu75** [comment]
@HarukiKozukapenguin 
Are files under `lib/M5AtomS3` public code? If so, please do not include in git commit and get automatically when build like as `add_custom_command` in cmake (FYI: https://docs.platformio.org/en/latest/scripting/custom_targets.html).

File: firmware/atom_s3_rosserial_message_display/src/main.cpp [hunk 1]
```diff
@@ -46,16 +174,15 @@ void loop()
...
   nh.spinOnce();
 
-  lcd.clear();
-  lcd.setCursor(0, 0);
+  M5.Lcd.clear();
+  M5.Lcd.setCursor(0, 0);
   if(!nh.connected())
     {
-      lcd.fillScreen(lcd.color565(255, 0, 0));
+    M5.Lcd.fillScreen(M5.Lcd.color565(255, 0, 0));
```
Comment (sugikazu75) [inline firmware/atom_s3_rosserial_message_display/src/main.cpp]: indent

File: firmware/atom_s3_rosserial_message_display/src/main.cpp [hunk 2]
```diff
@@ -20,24 +139,33 @@ void batteryVoltageCallback(const std_msgs::Float32& msg){
 
 ros::Subscriber<std_msgs::Float32> battery_voltage_sub_("battery_voltage_status", &batteryVoltageCallback);
 
+float maxVoltage = 16.8f; // 100% voltage
+float minVoltage = 14.8f; // 0%   voltage
+int batcell = 4;
```
Comment (sugikazu75) [inline firmware/atom_s3_rosserial_message_display/src/main.cpp]: Please do not use magic number which is specific value for each robots.

File: firmware/atom_s3_rosserial_message_display/src/main.cpp [hunk 3]
```diff
@@ -1,15 +1,134 @@
...
+    if(voltageRatio > float(k+1) / barNum)
+    {
+      color = M5.Lcd.color565(
+        (uint8_t)(255 - 255 * (k / float(barNum-1))),
+        (uint8_t)(255 * (k / float(barNum-1))), 0);
+    }
+    M5.Lcd.fillRoundRect(rect_x, rect_y, rect_w, rect_h, radius, color);
+  }
+}
+
```
Comment (sugikazu75) [inline firmware/atom_s3_rosserial_message_display/src/main.cpp]: these lines should be implemented other file

**Review by sugikazu75 — CHANGES_REQUESTED** [review]

**Comment by sugikazu75** [comment]
> @HarukiKozukapenguin Are files under `lib/M5AtomS3` public code? If so, please do not include git commit and get automatically when build like as `add_custom_command` in cmake (FYI: https://docs.platformio.org/en/latest/scripting/custom_targets.html).

I will add example to generate `ros_lib` automatically.

**Comment by sugikazu75** [comment]
@HarukiKozukapenguin  
This is an example to generate ros_lib automatically.  https://github.com/sugikazu75/riberry/pull/3

**Comment by HarukiKozukapenguin** [comment]
@sugikazu75 
I can get system that change battery cell by [this commit](https://github.com/sugikazu75/riberry/pull/2/commits/ad650176a431cbd563db1ca5d15530c84e5bc4b7)
(e.g. rosrun rosserial_python serial_node.py _port:=/dev/ttyACM0 __ns:=robot_ns **_bat_cell:=4**)

I also separate battery display class by making lib in lib directory ([this commit](https://github.com/sugikazu75/riberry/pull/2/commits/807e914d183677d37d345a7b17cb731a17dbb7a6))

File: firmware/atom_s3_rosserial_message_display/src/main.cpp (hunk 2 above)
Comment (HarukiKozukapenguin) [reply firmware/atom_s3_rosserial_message_display/src/main.cpp]: @sugikazu75
I can get system that change battery cell by [this commit](https://github.com/sugikazu75/riberry/pull/2/commits/ad650176a431cbd563db1ca5d15530c84e5bc4b7)
(e.g. _rosrun rosserial_python serial_node.py _port:=/dev/ttyACM0 __ns:=robot_ns **_bat_cell:=4**_)

File: firmware/atom_s3_rosserial_message_display/src/main.cpp (hunk 3 above)
Comment (HarukiKozukapenguin) [reply firmware/atom_s3_rosserial_message_display/src/main.cpp]: I also separate battery display class by making lib in lib directory ([this commit](https://github.com/sugikazu75/riberry/pull/2/commits/807e914d183677d37d345a7b17cb731a17dbb7a6))

**Comment by HarukiKozukapenguin** [comment]
> @HarukiKozukapenguin Are files under `lib/M5AtomS3` public code? If so, please do not include in git commit and get automatically when build like as `add_custom_command` in cmake (FYI: https://docs.platformio.org/en/latest/scripting/custom_targets.html).

I include m5stack/M5AtomsS3 file and add the file  in the command of _pio run -t upload_ ([commit](https://github.com/sugikazu75/riberry/pull/2/commits/237d2843bbf6408604cb51c52608dbb01a0088d6))

**Comment by sugikazu75** [comment]
@HarukiKozukapenguin 
This PR is ready to merge. Please test it with real machine (not necessary to flying test. just check LCD and real topic value).
`roslaunch robot_ns bringup.launch`
`rosrun rosserial_python serial_node.py _port:=tty**** __ns:=robot_ns`

**Comment by HarukiKozukapenguin** [comment]
@sugikazu75 
I tested with real machine, and it works` well. ([video](https://drive.google.com/file/d/1mCmK_9BThxoiokR91uYOh2mzOXKSEUvq/view?usp=sharing))

**Review by sugikazu75 — APPROVED** [review]

**Pull request merged**